test/ssltest_old.c

```diff
@@ -1891,7 +1891,7 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
          */
 
         /*
-         * We have non-blocking behaviour throughout this test program, but
+         * We have non-blocking behavior throughout this test program, but
```

Comment:
not being British, I'm okay with this change, but it might colour some other people's views :)